header.scss

```diff
@@ -0,0 +1,188 @@
+$pdsizehead:5px;
+$headclr:darkgrey;
+$brdrdet:.05px solid grey;
```

Comment:
.05 px really i don't think that precision is needed. I think it will be rounded off to 0 or 1px.